R/tm_g_response.R

```diff
@@ -156,7 +156,7 @@ tm_g_response <- function(label = "Response Plot",
                           ggplot2_args = teal.widgets::ggplot2_args(),
                           pre_output = NULL,
                           post_output = NULL,
-                          decorators = list(default = teal_transform_module())) {
+                          decorators = list()) {
```

Comment:
In other PRs this is set to NULL, and also a check for decorators is extended to NULL